spec/models/user_group_spec.rb

```diff
@@ -1,25 +1,28 @@
 require 'rails_helper'
 
-RSpec.describe UserGroup, :type => :model do
-  describe 'group groceries' do
-    context 'with groceries' do
+describe UserGroup, type: :model do
+  describe 'grocery scoping' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.